include/cantera/thermo/PengRobinsonMFTP.h

```diff
@@ -0,0 +1,429 @@
+//! @file PengRobinsonMFTP.h
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#ifndef CT_PENGROBINSONMFTP_H
+#define CT_PENGROBINSONMFTP_H
+
+#include "MixtureFugacityTP.h"
+#include "cantera/base/Array.h"
+
+namespace Cantera
+{
+/**
+ * Implementation of a multi-species Peng-Robinson equation of state
+ *
+ * @ingroup thermoprops
+ */
+class PengRobinsonMFTP : public MixtureFugacityTP
```

Comment:
I know this parallels the naming convention of RedlichKwongMFTP, but I'd like to suggest dropping the incomprehensible acronym suffix and go with either PengRobinsonPhase or just PengRobinson if there's no potential chance of confusion with another type of object in the future.

Reply:
We definitively decided to deprecate redundant pure fluid phases, yes?  I think the only useful part of the MFTP suffix is the "Mixture" part, but if we're in fact dropping the pure fluids, there would be no potential confusion, and I'd be find with just `PengRobinson`.

Reply:
Okay, MFTP is removed.